.github/workflows/build-and-test-windows.yml

```diff
@@ -10,6 +10,39 @@ env:
 
 jobs:
   windows-unittest:
+    strategy:
+      matrix:
+        group:
+          - receiver-1
+          - processor
+          - exporter
+          - extension
+          - internal
+          - other
+    runs-on: windows-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+      - name: Setup Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.17
+      - name: Setup Go Environment
+        run: |
+          echo "GOPATH=$(go env GOPATH)" >> $GITHUB_ENV
+          echo "$(go env GOPATH)/bin" >> $GITHUB_PATH
+        shell: bash
+      - name: Cache Go
+        uses: actions/cache@v3
+        with:
+          path: |
+            ~\go\pkg\mod
+            ~\AppData\Local\go-build
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+      - name: Run Unit tests
+        run: make gotest GROUP=${{ matrix.group }}
+
+  windows-unittest (receiver-0):
```

Comment:
Is this left for debug purposes?

Reply:
It was an attempt to have this job match the name style of the others. It appears not to have worked though, and I think I've found a better way anyways.